src/styles/home-button.less

```diff
@@ -0,0 +1,14 @@
+.sn-org-chart {
+  .sn-org-homebutton {
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    height: 20px;
+    width: 20px;
+    z-index: 1;
```

Comment:
Is this needed? Possible to just put it later in the DOM?

Reply:
seems possible!